### PR TITLE
rpm: fix E: incoherent-logrotate-file

### PR DESCRIPTION
**Pull request by kenhys**
It should be /etc/logrotate.d/PACKAGE-NAME.

### CODE REVIEW

File: td-agent/Rakefile [hunk 1]
```diff
@@ -324,7 +324,7 @@ class BuildTask
         configs = [
           "etc/td-agent/td-agent.conf",
           "etc/td-agent/td-agent.conf.tmpl",
-          "etc/logrotate.d/td-agent.logrotate",
+          "etc/logrotate.d/td-agent",
```
Comment (ashie) [inline td-agent/Rakefile:327]: The original td-agent package copies /etc/logrotate.d/td-agent.logrotate to /etc/logrotate.d/td-agent if it's not exist: https://github.com/treasure-data/omnibus-td-agent/blob/master/templates/package-scripts/td-agent/rpm/post#L45

So the patch is no problem if it doesn't replace the old one, but it seems that `noreplace` isn't specified in the spec file: https://github.com/clear-code/td-agent-builder/blob/d99d91f6c87fb9f6a3ba2de3fb0f9f45a1ad4ad9/td-agent/yum/td-agent.spec.in#L115

Comment (kenhys) [reply td-agent/Rakefile:328]: @ashie Thanks,
I've changed to add `noreplace` for it.

**Comment by ashie** [comment]
Thanks!

**Comment by ashie** [comment]
Oops, CI fails

```
RPM build errors:
    File not found: /root/rpmbuild/BUILDROOT/td-agent-3.7.1-1.el6.x86_64/etc/logrotate.d/td-agent.logrotate
Failed ./build.sh
```

**Comment by ashie** [comment]
It's fixed via #85

**Pull request merged**